segmentation/include/pcl/segmentation/impl/region_growing_rgb.hpp

```diff
@@ -735,22 +727,15 @@ pcl::RegionGrowingRGB<PointT, NormalT>::getSegmentFromPoint (pcl::index_t index,
     }
     // if we have already made the segmentation, then find the segment
     // to which this point belongs
-    for (auto i_segment = clusters_.cbegin (); i_segment != clusters_.cend (); i_segment++)
+    for (const auto& i_segment : clusters_)
     {
-      bool segment_was_found = false;
-      for (std::size_t i_point = 0; i_point < i_segment->indices.size (); i_point++)
-      {
-        if (i_segment->indices[i_point] == index)
-        {
-          segment_was_found = true;
-          cluster.indices.clear ();
-          cluster.indices.reserve (i_segment->indices.size ());
-          std::copy (i_segment->indices.begin (), i_segment->indices.end (), std::back_inserter (cluster.indices));
-          break;
-        }
-      }
-      if (segment_was_found)
+      const auto it = std::find (i_segment.indices.cbegin (), i_segment.indices.cend (), index);
+      if (it != i_segment.indices.cend())
```

Comment:
Perhaps, we can rewrite the condition to exit early
```cpp
for(loop) {
    const auto it = find();
    if (it == cend) { continue; }
    // rest of logic
    break;
}
```

Reply:
Using `if (it == cend) { continue; }` is more explicit, but could we just stay on the current version to make codes concise:blush:?